fe/fe-core/src/main/java/org/apache/doris/rpc/BackendServiceProxy.java

```diff
@@ -64,7 +64,7 @@ private static class Holder {
         }
 
         static BackendServiceProxy get() {
-            return proxies[count.addAndGet(1) % PROXY_NUM];
+            return proxies[Math.abs(count.addAndGet(1) % PROXY_NUM)];
```

Comment:
Why need `Math.abs`?

Reply:
If `count` exceeds the maximum value of `Integer`, the result of the remainder will be negative, which will cause an `IndexOutOfBoundsException` error